showcase/plot/scatterplot-large.js

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2016 - 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import React from 'react';
+
+import {
+  Hint,
+  HorizontalGridLines,
+  makeWidthFlexible,
+  MarkSeries,
+  VerticalGridLines,
+  XYPlot,
+  YAxis
+} from 'index';
+
+const FlexibleXYPlot = makeWidthFlexible(XYPlot);
+
+export default class Example extends React.Component {
+  componentWillMount() {
+    this.setState({
+      data: randomDatapoints(randomInt(700)),
```

Comment:
Why are you randomizing the number of points to render? Why not let that be a constant (but still randomize each datapoint)? Also, seems like a "scatterplot-large" would be significantly more than 700 points...I would expect a viz framework to be able to gracefully handle at least a couple thousand.